content/donate.adoc

```diff
@@ -0,0 +1,70 @@
+---
+layout: default
+title: "Donate to Jenkins"
+---
+
+NOTE: You can donate directly through the
```

Comment:
Why these random line breaks that will make future changes to sentences have a terrible PR diff?